enterprise/views.py

```diff
@@ -438,18 +440,23 @@ def post_course_specific_consent(self, request, course_id, consent_provided):
         except HttpClientError:
             raise Http404
 
-        enrollment_deferred = request.POST.get('enrollment_deferred')
```

Comment:
@brittneyexline I was also confused about the use of the `enrollment_deferred` POST parameter. I did not understand why we would ever NOT want to create an EnterpriseCourseEnrollment record here if one did not already exist. Perhaps @haikuginger has some background he could share?